modules/debts_api/lib/debts_api/v0/fsr_form_transform/enhanced_expense_calculator.rb

```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: true
+
+require 'debts_api/v0/fsr_form_transform/expense_calculator'
+
+module DebtsApi
+  module V0
+    module FsrFormTransform
+      class EnhancedExpenceCalculator
+        RENT = 'Rent'
+        MORTGAGE_PAYMENT = 'Mortgage payment'
+        FOOD = 'Food'
+
+        def initialize(form)
+          @form = form
```

Comment:
Instead of setting instance variables everywhere, could you use `attr_reader`s?

Reply:
I'm not sure I understand you. How would adding attr_readers negate the need to set instance variables?

Reply:
I suppose I should have said that instead of accessing instance variables directly, use Ruby's built-in getters via attr_reader. You'll still need to set them in your initializer.